crates/autopilot/src/on_settlement_event_updater.rs

```diff
@@ -198,6 +198,11 @@ impl Inner {
             .await;
 
             tracing::info!(?settlement, "settlement object");
+
+            if let Ok(settlement) = settlement {
+                let score = settlement.score();
+                tracing::info!(?score, "settlement score");
```

Comment:
Would it make sense to merge this log with the one at L200? so we have settlement and score in the same log, something like:

``rust
tracing::info!(?settlement, ?settlement.map(|settlement| settlement.score), "settlement object");
```

Reply:
sure, merged comments, afterall i believe only score will be observed... anyway temporary code so not super important